backend/http/http.go

```diff
@@ -161,6 +168,9 @@ func NewFs(ctx context.Context, name, root string, m configmap.Mapper) (fs.Fs, e
 	if !strings.HasSuffix(opt.Endpoint, "/") {
 		opt.Endpoint += "/"
 	}
+	if opt.GoIndexDriveIndex >= 0 {
+		opt.Endpoint += strconv.Itoa(opt.GoIndexDriveIndex) + ":/"
```

Comment:
Technically a drive index does not need to be specified, and if so should be determined by a test request to see redirects.